### DISCUSSION TITLE
Question: tw4449's Custom Mugs Download

### DISCUSSION THREAD

**Discussion by scohn** [Monika-After-Story/MonikaModDev#8413] [general - submod / external content]
I was wondering if there's somewhere here on Github where I can download the custom mugs tw4449 made for Monika. I already downloaded the custom rooms mod (awesome work btw, you're super talented looking forward to getting affection up to 400). I originally tried downloading off their google drive but it just never worked. I had no issues here on Github though.
It might have been in a plain sight and I'm just blind as a bat but if there's a link I could follow I'd really appreciate it, thank you.

**Comment by ThePotatoGuy** [comment]
Contact the creator on [reddit](https://www.reddit.com/user/tw4449/).
